src/metabase/automagic_dashboards/core.clj

```diff
@@ -583,7 +583,7 @@
   "Capitalize only the first letter in a given string."
   [s]
   (let [s (str s)]
-    (str (str/upper-case (subs s 0 1)) (subs s 1))))
+    (str (u/upper-case-en (subs s 0 1)) (subs s 1))))
```

Comment:
Are there cases where the first character of the string would be different? It would definitely be safe to just refer to str/ first with the ignore then change it later if we want. Thoughts? @metamben, @camsaul?

Reply:
BTW, why don't we just use `str/capitalize` instead of capitalize first? If it's for localization, wouldn't we want to use `u/upper-case-en`?

Reply:
Turkish has dotted and dotless variants for both cases: `i`, `İ` and `ı`, `I`. This means both `lower-case` and `upper-case` can be wrong.